kolibri/core/tasks/scheduler.py

```diff
@@ -173,7 +175,12 @@ def schedule(self, dt, func, interval=0, repeat=0, *args, **kwargs):
                 scheduled_time=naive_utc_datetime(dt),
                 saved_job=job.to_json(),
             )
-            session.merge(scheduled_job)
+            try:
+                session.merge(scheduled_job)
+            except ProgrammingError:
+                # needs to create job storage tables if they don't exist
+                job_storage.recreate_tables()
```

Comment:
This won't fix the issue, as it's only recreating the job storage, not the scheduled jobs table.

I think rather than trying to catch it at the time of erroring, we should do a startup sanity check like we do for Django migrations: https://github.com/learningequality/kolibri/blob/develop/kolibri/utils/sanity_checks.py#L96

We can then recreate both the `job_storage` and the `scheduled_jobs` tables if needed. Can then run this inside the intiialize method: https://github.com/learningequality/kolibri/blob/develop/kolibri/utils/main.py#L264